source/scone/input/keyboard_event.d

```diff
@@ -6,30 +6,28 @@ import scone.core.flags : hasFlag;
 
 struct KeyboardEvent
 {
-    this(SK key, SCK controlKey, bool pressed)
-    {
-        this.key = key;
-        this.controlKey = controlKey;
-        this.pressed = pressed;
-    }
-
     auto hasControlKey(SCK ck)
     {
         return controlKey.hasFlag(ck);
     }
 
     public SK key;
     public SCK controlKey;
-    public bool pressed;
+    version (Windows) public bool pressed = true;
```

Comment:
Unsure what I think about having keyboard events being pressed by default. I usually initialize everything in a falsy state.